src/ILLink.Tasks/ILLink.Tasks.csproj

```diff
@@ -71,10 +71,7 @@
 
     <MSBuild Projects="@(ProjectsToPublish)" Targets="Publish" />
     <ItemGroup>
-      <TfmSpecificPackageFile Include="$(PublishDir)/ILLink.*.dll" PackagePath="tools\$(TargetFramework)" />
-      <TfmSpecificPackageFile Include="$(PublishDir)/Mono.Cecil*.dll" PackagePath="tools\$(TargetFramework)" />
-      <TfmSpecificPackageFile Include="$(PublishDir)/illink.dll" PackagePath="tools\$(TargetFramework)"
-                              Condition=" '$(TargetFramework)' == 'netcoreapp2.0' " />
+      <TfmSpecificPackageFile Include="$(PublishDir)/*.dll" PackagePath="tools\$(TargetFramework)" />
```

Comment:
The specific includes were there originally to avoid including `System.Diagnostics.DiagnosticsSource.dll` and `illink.dll` on `net472`. This is no longer necessary for `illink.dll` because the packagereference to it is conditional on the tfm, but with this change `System.Diagnostics.DiagnosticsSource.dll` will also redistributed for `net472`. I don't actually understand what this dll is for or why it is getting published. @nguerrera, do you know what's the right thing to do with it?

Reply:
Can you send me project.assets.json for this project and a binlog of the publish?

Reply:
I explicitly excluded assets from the `DiagnosticSource` package below. Excluding assets from `NETStandard.Library` in `ILLink.Tasks.csproj` wasn't sufficient, because the assets were coming from an indirect dependency of that package.

Reply:
I'll also try to move https://github.com/mono/linker/pull/471 forward so that we use a `netstandard2.0` version of cecil, which won't have the same problem.